src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/services/RestrictedPatientsService.kt

```diff
@@ -42,24 +43,42 @@ class RestrictedPatientsService(
       commentText = dischargeToHospital.commentText
     )
 
-    return transform(restrictedPatientsRepository.save(restrictedPatient))
+    return transform(
+      restrictedPatientsRepository.save(restrictedPatient),
+      response?.restrictivePatient?.supportingPrison,
```

Comment:
Just checking - is this intended to be the supporting prison (the same as below)? If so, not sure there is much point having both bits of data.

Reply:
I'm still not sure if they can be different yet. What I should of done here is make a request to pull the data from /agencies/type rather than reusing the supporting one. I'll tidy it up in the next PR